backend/README.md

```diff
@@ -63,27 +63,86 @@ The following steps are optional and are only required if you need to update the
 
 By completing these steps, the backend will be primed with the essential verifiers for its tasks.
 
-## Examples
+## Summa solvency flow example
 
-### Running the Inclusion Verification
+This example illustrates how Summa interacts with the Summa contract and the user side.
 
-This example demonstrates how a user can verify the inclusion of their account in the Merkle Sum Tree. 
-In this example, the CEX provides the user with their `balances` and `username`, but not the `leaf_hash`. 
+To execute this example, use the command:
+
+```
+cargo run --release --example summa_solvency_flow
+```
+
+### 1. Submitting Address Ownership to the Summa Contract
+
+First, we submit proof of address ownership to the Summa contract. This is a critical step to register these proofs on-chain, facilitating the validation of asset ownership within Summa.
+
+Key points:
+
+- An instance of `AddressOwnership`, named `address_ownership_client`, is initialized with the `signatures.csv` file, which contains the signature data.
+
+- The `dispatch_proof_of_address_ownership` function sends a transaction to the Summa contract to register CEX-owned addresses.
+
+- After dispatching the transaction, the example computes the hashed addresses (address_hashes) to verify they've been correctly registered in the Summa contract
 
-The user will generate the `leaf_hash` themselves and then verify its inclusion in the tree.
 
-Make sure you have the required files:
-- `backend/ptau/hermez-raw-11`
-- `backend/src/apis/csv/assets.csv`
-- `zk_prover/src/merkle_sum_tree/csv/entry_16.csv`
+Note: This demonstration takes place in a test environment. In real-world production, always ensure that the Summa contract is correctly deployed on the target chain.
 
+If executed successfully, you'll see:
 
-To run the example:
 ```
-cargo run --example verify_inclusion
+1. Ownership proofs are submitted successfully!
 ```
 
-On successful execution, you'll observe a message indicating the verification outcome:
+
+### 2. Submit Proof of Solvency
```

Comment:
Can you specify that the goal of the proof of solvency is also to check that the assets controlled by the exchange are greater than the liabilities (as committed in the Merkle Sum Tree Root Hash)